docs/user-guide/authentication-providers-for-apiml.md

```diff
@@ -0,0 +1,39 @@
+# Configuring an authentication provider for API Mediation Layer
+
+:::info Required roles: system administrator, security administrator
+:::
+
+Choose from the following providers to handle authentication for the API Gateway:  
+* [z/OSMF Authentication Provider](#z-osmf-authentication-provider)
+* [SAF Authentication Provider](#saf-authentication-provider)
+
+:::note
+For development purposes, a dummy authentication provider is also available. This provider is not intend for production purposes.
```

Comment:
The initial version of the file is here: https://github.com/zowe/docs-site/blob/janan07-create-auth-providers-for-apiml-section/docs/extend/extend-apiml/api-mediation-layer-development-setup.md